awsx/package.json

```diff
@@ -25,7 +25,7 @@
   "//": "Pulumi sub-provider dependencies must be pinned at an exact version because we extract this value to generate the correct dependency in the schema",
   "dependencies": {
     "@pulumi/aws": "6.65.0",
-    "@pulumi/docker": "4.5.1",
+    "@pulumi/docker-build": "0.0.8",
```

Comment:
Is this possible a breaking change? Like if I pull in `p/docker` indirectly via `awsx` and then reference it from my code, my imports would break?

I don't know if that's what would actually happen or if it's avoidable. Would you leave in the old p/docker dependency even if it's unused?

Mostly just asking to make sure we've thought through any edge cases.

Reply:
IMO it's not a breaking change. Users shouldn't depend on transitive dependencies directly. They should add those to their own dependencies.

That being said, I'm currently workin on another awsx feature that'll add back the docker provider for push without build scenarios so we're gonna be covered here